src/System.Threading.Tasks.Extensions/src/System.Threading.Tasks.Extensions.csproj

```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'" Include="System.Runtime.CompilerServices.Unsafe" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Condition="'$(IsPlaceholderTargetFramework)' != 'true'" />
```

Comment:
Related to the question above: Is this not giving you trouble as it seems that TFI is also including netstandard? (If I am reading this correctly)

Reply:
Nope. This applies to the non placeholder frameworks which includes NETStandard.